src/main/java/chatbot/gui/Gui.java

```diff
@@ -0,0 +1,68 @@
+package chatbot.gui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.VBox;
+
+import chatbot.ChatBot;
```

Comment:
I believe that the import structure can be restructured to the correct lexicographical order for 'chatbot.ChatBot' import. It should be before 'javafx.scene.layout.VBox'.

Reply:
iirc checkstyle gave me warning for putting package imports before java modules, but thanks for your suggestion!